### PR TITLE
[ESP32][PWM] Make PWM work on ESP32

### PR DESCRIPTION
**Pull request by clumsy-stefan**
Make PWM work on ESP32 modules. See issue letscontrolit#3966

### CODE REVIEW

File: src/src/Helpers/Hardware.cpp [hunk 1]
```diff
@@ -1607,7 +1607,7 @@ void initAnalogWrite()
   #if defined(ESP32)
   for(uint8_t x = 0; x < 16; x++) {
     ledChannelPin[x] = -1;
-    ledChannelFreq[x] = ledcSetup(x, 0, 10); // Clear the channel
+    ledChannelFreq[x] = ledcSetup(x, 1000, 10); // Clear the channel
```
Comment (TD-er) [inline src/src/Helpers/Hardware.cpp:1610]: So this is the actual fix?

Comment (clumsy-stefan) [reply src/src/Helpers/Hardware.cpp:1610]: No, only one part of it, see other commits. Only when I change all three in Hardware.cpp and the one in Network.cpp it works.

Comment (clumsy-stefan) [reply src/src/Helpers/Hardware.cpp:1610]: I just don't see why `pinMode(pin, OUTPUT);` breaks it. As well for GPIO -> LED in HW (Network.cpp) as well as with just the pwm command (Hardware.cpp).

**Comment by TD-er** [comment]
All implemented by: https://github.com/letscontrolit/ESPEasy/pull/4030

**Pull request closed without merging**